### PR TITLE
Feature/manual controls

### PR DESCRIPTION
**Pull request by duongel**
added hotkeys for manual brightness incrementation and decrementation

### CODE REVIEW

File: Lunar/AppDelegate.swift [hunk 1]
```diff
@@ -501,6 +520,23 @@ class AppDelegate: NSObject, NSApplicationDelegate, CLLocationManagerDelegate, N
...
+    private func increaseBrightness(_ amount: Int8 = 2) {
+        adjustBrightness(amount)
+    }
+    
+    private func decreaseBrightness(_ amount: Int8 = 2) {
+        adjustBrightness(-amount)
+    }
+    
+    private func adjustBrightness(_ amount: Int8) {
+        let currentBrightness = brightnessAdapter.displays.first?.value.brightness.intValue
```
Comment (duongel) [inline Lunar/AppDelegate.swift]: `brightnessAdapter.displays.first?.value.brightness.intValue` does not seem to return the display's current brightness reliably. Any ideas why or which function to use instead?

**Comment by alin23** [comment]
Hi @duongel !

Thank you for the PR! This is something I've been wanting to implement after my current trip.

There a few things that we should decide upon before merging this:
* Are the default hotkeys particular enough so that they don't interfere with other system/app hotkeys?
  - I've seen people ask for this functionality bound to the brightness keys instead of other hotkeys, what do you think about this idea?
* By using `setLightPercent()`, the brightness/contrast will be confined between the per-monitor min/max limits. Do we want this or should we override the min/max limits when manually adjusting the brightness?

**Comment by alin23** [comment]
Thanks for the prompt follow-up @duongel 

Yes, the <kbd>CTRL</kbd>+<kbd>F1</kbd> and <kbd>CTRL</kbd>+<kbd>F2</kbd> would be ok for most users. I think we should make this configurable in the near future to make everyone happy, but for now I like it this way.

You're making a good point about the min/max limits, I was thinking the same 😄 

There's no way to get the brightness reliably as far as I know because the DDC protocol isn't fully implemented in a lot of monitors. I tried everything here but there doesn't seem to be a way to get read access to these properties. 

The value you're getting here `brightnessAdapter.displays.first?.value.brightness.intValue` is the value we are setting, and most of the time we are not saving it in the DB because it is not that useful/reliable.

**Comment by alin23** [comment]
I like the code but I can't test it right now because I'm still on the road with no access to a monitor. I think on Monday I'll get home and after I test the feature I'll make a final review or I will merge it right away depending on how it works.

Good work and many thanks for taking the time to share it! 💯 
You'll make a bunch of users very happy 😄

**Pull request merged**